packages/frontend/src/header/Header.tsx

```diff
@@ -0,0 +1,11 @@
+import './Header.scss';
+
+export const Header = () => {
+  return (
+    <header className="App-header">
+        <p>
+          <a href='#TreeCounter'>Tree Counter</a> | <a href='#About'>About</a> | <a href='#Map'>Map</a>
+        </p>
+    </header>
+  );
+};
```

Comment:
This React looks perfect to me. You're using functional components and returning JSX. There's no state or business logic so it's just a UI component which is completely fine.